src/main/java/jag/Parser.java

```diff
@@ -0,0 +1,75 @@
+package jag;
+
+/**
+ * Parser Class is responsible for taking in a user input and making sense of it
+ * by checking what the user has entered and returning the right type of command
+ * that the user has requested
+ */
+public class Parser {
+
+    /**
+     * Responsible for parsing a user input to read it's command
+     * and return the appropriate instance of the right command
+     *
+     * @param fullCommand String representation of the full user input
+     * @return An instance of a type of Command, from the list of Commands
+     *              that has been extended by the abstract class Command
+     * @throws AExceptions Instance thrown in the event user input does not include
+     *              any of the right commands available
+     */
+    public static Command parse(String fullCommand) throws AExceptions {
+        // Breaking down command input
+        String[] splitWords = fullCommand.split(" ");
+        String com = splitWords[0].toUpperCase();
+        Commands command;
```

Comment:
Perhaps this command variable could be replaced with commands, to show how it represents multiple enums.
```suggestion
        Commands commands;
```